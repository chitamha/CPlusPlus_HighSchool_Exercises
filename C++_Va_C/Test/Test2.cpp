#include <bits/stdc++.h>
#define fi first
#define se second
#define llll pair<ll, ll>
#define ll long long
#define maxn 1000005
using namespace std;

int main(){
    freopen("TEST2.INP", "r", stdin);
    freopen("TEST2.OUT", "w", stdout);
    ios_base::sync_with_stdio(false);
    cin.tie(nullptr); cout.tie(nullptr);

    return 0;
}
