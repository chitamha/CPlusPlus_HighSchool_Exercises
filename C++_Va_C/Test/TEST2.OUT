4 9 8
445 12 8
713 9 8
753 9 8
944 9 8
960 9 8

